sonar-plugin-api/src/main/java/org/sonar/api/issue/filter/IssueFilterChain.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * SonarQube, open source software quality management tool.
+ * Copyright (C) 2008-2014 SonarSource
+ * mailto:contact AT sonarsource DOT com
+ *
+ * SonarQube is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * SonarQube is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.api.issue.filter;
+
+/**
+ * A filter chain is an object provided to issues filters for fine control over the filtering logic. Each filter has the choice to:
+ * <ul>
+ *  <li>Accept the issue</li>
+ *  <li>Reject the issue</li>
+ *  <li>Let downstream filters decide by passing the issue to the rest of the chain</li>
+ * </ul>
+ * 
+ * @since 5.3
+ */
+public interface IssueFilterChain {
+    /**
+     * Called by a filter to let downstream filters decide the fate of the issue
+     */
+    boolean accept(FilterableIssue issue);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this line start at column 3. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AIndentationCheck)